src/components/Card/Card.js

```diff
@@ -0,0 +1,21 @@
+import useStore from '../useStore/useStore';
+import { CardContainer } from '../Form/Form.styled';
+
+export default function Card() {
+	const newBook = useStore(state => state.newBook);
+	return (
+		<ul>
+			{newBook.map(book => {
+				return (
+					<CardContainer key={book.id}>
+						<li>
+							<p>Author: {book.author}</p>
+							<p>Title: {book.title}</p>
+							<p>{book.comment}</p>
+						</li>
+					</CardContainer>
+				);
+			})}
+		</ul>
+	);
+}
```

Comment:
Consider renaming your "newBook" state to something that fits better. newBook seems to be an array of books, so why not calling it "books" or "newBooks"?

Reply:
Changed